src/scss/components/file/_index.scss

```diff
@@ -0,0 +1,5 @@
+@import "components/form/config";
+@import "components/box/config";
+
+/// ========== MIXINS/PARTIALS ========== ///
+@import "config";
```

Comment:
This file should only import the config and partials.

* Begin each import path with `./`

```scss
@import "./config";

@import "./hx-file-control";
@import "./hx_drop-fence";
@import "./hx_drop-zone";
@import "./hx-file-icon";
@import "./hx-file-tile";
```

Reply:
Updated import order.